rotkehlchen/tests/api/test_nfts.py

```diff
@@ -150,29 +152,92 @@ def test_nft_ids_are_unique(rotkehlchen_api_server):
 
 @requires_env([TestEnvironment.NIGHTLY, TestEnvironment.NFTS])
 @flaky(max_runs=3, min_passes=1)  # all opensea calls have become quite flaky
-@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4]])
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4, TEST_ACC5]])
 @pytest.mark.parametrize('start_with_valid_premium', [True])
 @pytest.mark.parametrize('ethereum_modules', [['nfts']])
 def test_nft_balances_and_prices(rotkehlchen_api_server):
```

Comment:
For what I saw there is none, can you create a test mocking the open sea api and chaning the image and price of an nft and verify that it gets updated. Also removing one from the response and seeing that it gets deleted from the local cache. No need to query opensea, you can just mock the responses